### ISSUE TITLE
Find and replace dialog: Alt+L to off: Selection lost

### ISSUE THREAD

**Issue by ZtrevorZ** [Microsoft/vscode#50567]
When replacing text (control + h to  open the find and replace dialog): I would like to toggle "Find in selection" to **off** with Alt+L but keep my selection. Currently when you have valid "before" and "after" entries in the dialog (as opposed to blanks) and then press Alt+L to "off", the text is deselected and you have to re-select it by hand. Finding only (without replace) seems to work OK ... it is when replacing text that the problem occurs. Thanks!
- VSCode Version: 1.23.1
- OS Version: x64 (Windows 10 64bit)

**Comment by rebornix** [comment]
@ZtrevorZ based on what I tested on latest Insiders (we made some fixes this month), if we have `editor.autoFindInSelection` turned on, when we open the find widget with either ctrl+f and ctrl+h after selecting some text, the focus will be in the find input box. Since our find widget does searches incrementally, the moment when you switch off "Find In Selection" , we will research the keyword in the find input box in the whole document, which will finally move the selection to the nearest match.

As I no longer reproduce what you described and our workflow is described as above, I'd like to close it for now.
